EIPS/eip-X.md

```diff
@@ -0,0 +1,220 @@
+---
+eip: <to be assigned>
+title: Cross-Chain Execution
+description: This specification defines an interface that supports execution across EVM networks.
+author: Brendan Asselstine (@asselstine), Anna Carroll (@anna-carroll), Hadrien Croubois (@Amxx), Theo Gonella (@mintcloud), Rafael Solari (@rsolari), Auryn Macmillan (@auryn-macmillan), Nathan Ginnever (@nginnever)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): ERC
+created: 2022-06-14
+---
+
+# EIP-?: Cross-Chain Execution
+
```

Comment:
```suggestion
```
Must follow the template layout: https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md